### ISSUE TITLE
Link to contributors in some other/better way

### ISSUE THREAD

**Issue by peterbe** [mdn/stumptown-content#117]
Please be aware of this discussion: https://github.com/mozilla/kuma/issues/5717
I think stumptown-renderer would equally benefit from not having to render that massive list of names and URLs. 

This issue belongs both *and* either here and in [stumptown-renderer](https://github.com/mdn/stumptown-renderer) but they're intertwined. 

Content in stumptown will almost certainly "always" (at least of URIs that existed on the Wiki) be a mix of historical contributions from the Wiki as well as git commits within a directory. 

We will have to port the contributions from the Wiki into stumptown-content. [Like we've already done](https://github.com/mdn/stumptown-content/blob/master/content/html/reference/elements/video/contributors.md)
We can union this by doing some fancy `git log` commands [0] as time progresses the list of contributions will be a little bit of Wiki historical and a little bit `git log`. 

One big question; if we take head from https://github.com/mozilla/kuma/issues/5717 , especially in point-of-view of stumptown-renderer, if you don't display the full list of names, where else do you display it? 

[0] E.g.
```bash
git log -M --follow --pretty=tformat:"%aN %aE" -- content/html/reference/elements/video/docs.md | uniq
```

**Comment by joedarc** [comment]
> Just need to figure out how the heck to map full names and emails down to GitHub usernames.

Yeah, my user is actually a rough test case for this as well because my username is joedarc. Might be able to use the [GitHub API](https://developer.github.com/v3/repos/commits/) to retrieve the info you need?

`GET /repos/:owner/:repo/commits?path=:path-to-file`
